pkg/resources/collector/templates.go

```diff
@@ -101,6 +102,53 @@ journald/k3s:
   units: [ "k3s" ]
   directory: {{ . }}
 `))
+
+	templateKubeAuditLogs = template.Must(template.New("kubeauditlogsreceiver").Parse(`
+filelog/kubeauditlogs:
+  include: {{ . }}
+  start_at: beginning
+  include_file_path: false
+  include_file_name: false
+  operators:
+  - type: json_parser
+    id: parse-body
+    timestamp:
+      parse_from: attributes.stageTimestamp
+      layout: '%Y-%m-%dT%H:%M:%S.%LZ'
+  - type: add
+    field: attributes.log_type
+    value: controlplane
```

Comment:
I don't think we should tag these as controlplane as that has implications for the controlplane ai models

Reply:
Changed to `auditlog`.